Netable/Netable/Netable.swift

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2020 Steamclock Software. All rights reserved.
 //
 
-import Combine
+@preconcurrency import Combine
```

Comment:
I'm on the fence about if we should include this condition or not

Reply:
Seems like we shouldn't, unless we have to, in which case we should 😄 

Is it generating a bunch of extra warnings if we don't do that? I would have hoped that Apple frameworks would have been made concurrency aware, particularly since I think there is a tiny bit of code using concurrency in Combine (there's at least a method that creates an AsyncSequence from a publisher). If it has been updated, that sort of implies that any remaining warnings are genuine.

Reply:
It adds two extra warnings, I'm just gonna remove it for now and we can live with the warnings.